packages/rangelink-core-ts/src/external-deps/couimet/dynamic-testing/uniqueFilePath.ts

```
@@ -0,0 +1,193 @@
+    const ext = extension !== undefined ? normalizeExtension(extension) : getUniqueFileExtension();
+    return { value: `${basename}${ext}`, isExplicit: true };
+  }
+
+  // Auto-generated
+  const ext = extension !== undefined ? normalizeExtension(extension) : getUniqueFileExtension();
+
+  if (!unique) {
+    if (maxLength !== undefined && maxLength <= ext.length) {
+      throw new Error(
+        `${MAXLENGTH_TOO_SMALL}: maxLength (${maxLength}) too small for filename (extension alone = ${ext.length} chars)`,
+      );
```

ESLint

[error] Replace `⏎········`${MAXLENGTH_TOO_SMALL}:·maxLength·(${maxLength})·too·small·for·filename·(extension·alone·=·${ext.length}·chars)`,⏎······` with ``${MAXLENGTH_TOO_SMALL}:·maxLength·(${maxLength})·too·small·for·filename·(extension·alone·=·${ext.length}·chars)``

(prettier/prettier)